src/hooks/trigger.ts

```diff
@@ -52,6 +78,7 @@ export type Condition<H extends HookContext, T = Record<string, any>> =
   | ((item: T, context: H) => Promisable<boolean>);
 
 export interface Subscription<
+  IsBatchMode extends boolean,
```

Comment:
Shouldn't `IsBatchMode` be `false` by default (`IsBatchMode extends boolean = false`)?